prow/tide/tide_test.go

```diff
@@ -726,8 +756,14 @@ func TestDividePool(t *testing.T) {
 			if pj.Spec.Type != prowapi.PresubmitJob && pj.Spec.Type != prowapi.BatchJob {
 				t.Errorf("PJ with bad type in subpool %s: %+v", name, pj)
 			}
-			if pj.Spec.Refs.Org != sp.org || pj.Spec.Refs.Repo != sp.repo || pj.Spec.Refs.BaseRef != sp.branch || pj.Spec.Refs.BaseSHA != sp.sha {
-				t.Errorf("PJ in wrong subpool. Got PJ %+v in subpool %s.", pj, name)
+			referenceRef := &prowapi.Refs{
+				Org:     sp.org,
+				Repo:    sp.repo,
+				BaseRef: sp.branch,
+				BaseSHA: sp.sha,
+			}
+			if diff := deep.Equal(pj.Spec.Refs, referenceRef); diff != nil {
```

Comment:
Does reflect.DeepEqual not work here?

Reply:
It does work, but `deep` provides a very easy to read diff which is super helpful when debugging issues with the tests